src/main/java/io/icker/factions/api/persistents/Faction.java

```diff
@@ -84,6 +84,11 @@ public Faction(
 
     public Faction() {}
 
+    public ArrayList<Relationship> getRelationships() {
```

Comment:
Can this return a stream? It seems like every time it is used it is turned into a stream, so there is no reason to collect it back to a list.